Empty spaces

Xnst opened the pull request:
In salome sometimes you create empty sets in your .unv file. This change omits the empty sets in the inp-file and unv2ccx does not crash.

Xnst committed:
Fixed reading empty groups and skipping them

Xnst committed:
Fixed reading empty groups and skipping them

imirzov commented:
Thank you for digging the code!
Sorry, I don't like your line 143 of the UNVParser.py :)) Please, use function 'Trim Trailing Whitespace' in your code editor.
Now line 126. What if parsed file will have more than one empty line? What if empty lines are present in the parsed file everywhere?

imirzov commented:
Please, try another attempt. I'll wait for your next request.
If you'll give up - please, attach file you're trying to convert - I'll modify the code for your needs.

Xnst committed:
More robust reading of empty sets in dataset 2467 section

Xnst commented:
Hi Ihor,

I changed the reading of the empty rows down to where it should be. Also, I removed the white spaces. My editor just loves to add them in empty lines :) .

I hope this is a more robust version and more dedicated to solve the problem when reading empty sets from a salome dump of a unv-file.

I pasted the 2467 section of a unv-file that comes from salome. so you can see for your self. The problem occurs when you create a group in salome and do not have any in it (a 0 in the last position in the group definition row).

And to be honest, this is my first pull request and I am just trying out the workflow. Sorry if there might be some unorthodox usage from my side. First I replied to the mail notification I got, but .. I think this i sthe proper way to do it.

Regards
Niclas

```
    -1
  2467
         1         0         0         0         0         0         0         0
nod1

         2         0         0         0         0         0         0         1
nod1
         7         2         0         0
    -1
```

imirzov commented:
Thank you, Niclas! I'll test your version and merge, but later. Too busy now, sorry.
